packages/editor-ui/src/mixins/externalHooks.ts

```diff
@@ -1,29 +1,31 @@
-import { IExternalHooks, IRootState } from '@/Interface';
-import { store } from '@/store';
+import { IExternalHooks } from '@/Interface';
 import { useWebhooksStore } from '@/stores/webhooks';
 import { IDataObject } from 'n8n-workflow';
 import { Store } from 'pinia';
 import Vue from 'vue';
 
+declare global {
+	interface Window {
+		n8nExternalHooks: Record<string, Record<string, Array<(store: Store, metadata?: IDataObject) => Promise<void>>>>;
```

Comment:
should be optional.. does not always get set

Reply:
done